.github/workflows/check-shell.yml

```diff
@@ -10,3 +10,5 @@ jobs:
         uses: ludeeus/action-shellcheck@master
         with:
           ignore: Vagrantfile
+        env:
+          SHELLCHECK_OPTS: -e SC2016
```

Comment:
@fsimonis I would prefer that we don't globally disable rules. Better add

```
# shellcheck disable=SC2016
```

above the respective line.

Reply:
Or I just do that directly.